.github/workflows/dev-cluster-check.yml

```diff
@@ -83,6 +83,7 @@ jobs:
           working-directory: integration-tests
       - name: Archive junit test results
         uses: actions/upload-artifact@v3
+        if: ${{ always() }}
```

Comment:
I think we don't need curly braces here, wdyt?

Reply:
Seems, so, though it is used in examples
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsif

Reply:
we have a mix in our workflows, some with curly [braces](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/dev-cluster-check.yml#L109) and some without [them](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/.github/workflows/dev-cluster-check.yml#L117)

Reply:
Good point. I can look at it and try to unify this in the separate PR.